modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc

```diff
@@ -36,17 +36,20 @@ metadata:
   annotations:
     che.eclipse.org/oauth-scm-server: github
     che.eclipse.org/scm-server-endpoint: __<github_server_url>__ <2>
-    che.eclipse.org/scm-github-disable-subdomain-isolation: true <3>
```

Comment:
can't we simply set default to `false` to avoid confusion?

Reply:
done